ansible/local_testing/README.md

```diff
@@ -37,12 +61,25 @@ vms:
 
 # Fixes/Notes
 
+
+### Ansible cannot decrypt the files encrypted with ansible vault.
+
+* The `ansible\roles\certbot\vars\main.yaml` and `ansible\roles\pydis-users\vars\main.yaml` files have been encrypted with ansible vault due to their sensitive content.
+
+* If you're not part of the people with access to the vault, you're going to have to either define your own variables or just exclude the appropriate roles.
```

Comment:
Maybe we can make this a bit more concise:
```suggestion
* If you lack access to the vault, you're going to have to either define your own variables or exclude the appropriate roles.
```

Reply:
Simple & elegant, granted